azure-pipelines.perf-test.yml

```diff
@@ -10,86 +10,132 @@ variables:
-  # @fluentui/perf-test-northstar Baseline
-  - script: |
-      yarn perf:test:base
-    condition: eq(variables.isPR, false)
-    workingDirectory: packages/fluentui/perf-test-northstar
-    displayName: Perf Test Base (Fluent)
-
-  # @fluentui/perf-test-northstar PR
-  - script: |
-      yarn perf:test
-    condition: eq(variables.isPR, true)
-    workingDirectory: packages/fluentui/perf-test-northstar
-    displayName: Perf Test (Fluent)
-
-  # Fabric: perf-test will build, bundle, and run-perftest
-  - script: |
-      yarn just perf-test
-    workingDirectory: apps/perf-test
-    displayName: Perf Test
-
-  - task: AzureUpload@2
-    displayName: Upload Perf Test Result to PR deploy site (Fluent N*)
-    inputs:
-      SourcePath: 'packages/fluentui/perf-test-northstar/dist'
-      azureSubscription: $(azureSubscription)
-      storage: $(azureStorage)
-      ContainerName: '$web'
-      BlobPrefix: '$(deployBasePath)/perf-test-northstar'
-
-  - task: AzureUpload@2
-    displayName: Upload Perf Test Result to PR deploy site (Fluent)
-    inputs:
-      SourcePath: 'apps/perf-test/dist'
-      azureSubscription: $(azureSubscription)
-      storage: $(azureStorage)
-      ContainerName: '$web'
-      BlobPrefix: '$(deployBasePath)/perf-test'
-
-  - task: GithubPRComment@0
-    displayName: 'Post Perf Results to Github Pull Request'
-    inputs:
-      githubOwner: microsoft
-      githubRepo: 'fluentui'
-      blobFilePath: '$(Build.SourcesDirectory)/$(PerfCommentFilePath)'
-      status: '$(PerfCommentStatus)'
-      uniqueId: 'perfComment9423'
-
-  - script: |
-      yarn stats:build
-    condition: eq(variables.isPR, false)
-    displayName: Bundle Statistics (master only)
-
-  - script: |
-      yarn perf
-    condition: eq(variables.isPR, false)
-    displayName: Performance Tests (master only)
-
-  # HEADS UP: also see tag-version-prefix in fluentui-publish.js
-  - script: |
-      yarn stats:save --tag=`git tag --points-at HEAD | grep ^@fluentui/react-northstar_v | grep -o 'northstar_v.*'`
-    condition: eq(variables.isPR, false)
-    displayName: Save Statistics to DB (master only)
-    env:
-      STATS_URI: $(STATS_URI)
-
-  - template: .devops/templates/cleanup.yml
+jobs:
+  - job: PerfTestNorthstar
+    displayName: Perf test for @fluentui/react-northstar
+    workspace:
+      clean: all
+    steps:
+      - template: .devops/templates/tools.yml
+
+      - task: Bash@3
+        inputs:
+          filePath: yarn-ci.sh
+        displayName: yarn
+
+      - script: |
+          yarn just check-if-package-affected --packageAffected @fluentui/react-northstar
+        workingDirectory: packages/fluentui/perf-test-northstar
+        displayName: Check if @fluentui/react-northstar was modified
+
+      # Build @fluentui/perf-test-northstar
+      - script: |
+          yarn lage build --to @fluentui/perf-test-northstar --to @fluentui/react-conformance --verbose
```

Comment:
Why does it need react-conformance?

Reply:
northstar needs it to build successfully. This wasn't an issue before because `react-conformance` gets built when v8 got built